server/grpc_service.go

```diff
@@ -1835,6 +1836,9 @@ const globalConfigPath = "/global/config/"
 // Since item value needs to support marshal of different struct types,
 // it should be set to `Payload bytes` instead of `Value string`
 func (s *GrpcServer) StoreGlobalConfig(_ context.Context, request *pdpb.StoreGlobalConfigRequest) (*pdpb.StoreGlobalConfigResponse, error) {
+	if s.client == nil {
```

Comment:
Do we need to pick it to other branches?

Reply:
yes, due to this pr, seems like we need to change a lot...
https://github.com/tikv/pd/pull/4308/files#diff-eb404b45d0a569b05f58f4e4bf2b42b32150be75ec2ed72f86330c4b5d739eab